src/models/RootStore.ts

```diff
@@ -0,0 +1,28 @@
+import {Instance, types} from 'mobx-state-tree';
+import {createContext} from 'react';
+
+const store = types
+  .model('RootStoreModel')
+  .props({
+    count: types.number,
+  })
+  .views(self => ({
+    get Count(): number {
+      return self.count;
+    },
+    set Count(val: number) {
```

Comment:
If we are incrementing and decrementing using actions, then why do we need this?